packages/raml-generator/src/main.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
+
+function generateRaml(options) {
+  const resolvedSourcePath = path.resolve(options.apiSpecSourcePath);
+
+  // Check if the spec file exists.
+  fs.access(resolvedSourcePath, fs.constants.F_OK, err => {
+    if (err) {
+      console.error(
+        `${resolvedSourcePath} does not exist`,
+        chalk.red.bold('ERROR')
+      );
+      process.exit(1);
+    }
+
+    generateRamlIfSpecPath({
+      ...options,
+      apiSpecSourcePath: resolvedSourcePath,
+    });
+  });
+}
+
+function generateRamlIfSpecPath(options) {
+  const resolvedDestinationPath = path.resolve(options.apiSpecDestinationPath);
+
+  console.log(
+    `RAML docs will be generated in this directory: ${chalk.blue(
+      resolvedDestinationPath
+    )}`
+  );
+
+  const jarFile = path.resolve(
+    __dirname,
+    '../jar/cli-application-1.0.0-20200221085820-all.jar'
```

Comment:
Why is this hardcoded??

Reply:
Because the location is consistent.

Reply:
What I mean is that the file name should be read programmatically and not hardcoded. This is error prone as if the jar file name changes, and you forget to update it here, the script won't work anymore.

```js
const fileName = fs.readdirSync(path.join(__dirname, '../jar'))
  .find(name => name.startsWith('cli-application'));
```

Reply:
Thanks for explaining, but your suggestion is still prone to the same error you mentioned from my perspective. But I'll make the changes as you want it.

Reply:
The code snippet was just an example to illustrate what I meant.
I don't know how the process to update the jar file is, so the final implementation should be based on that. I would assume that only 1 jar is included in the package.
The point is to make the updates to the jar file to not break the script.

Reply:
Yes, there is only one jar file.

To update the jar file, manually downloading and copy-paste is necessary. I guess one benefit of manual download is that we can rename the file to exactly what we want and then keep that naming consistent.

Reply:
Yep, that is another option